### DISCUSSION TITLE
wiki/guide?

### DISCUSSION THREAD

**Discussion by thecloudking** [SigmundGranaas/forgero#787] [General]
im having some problems getting started and knowing where to go moving forward. is there a guide or a short wiki somewhere? the github wiki seems to be mostly technical about how the mod is coded, not actually helpful for playing with it

**Comment by Steveplays28** [comment]
Hi thecloudking, there should be a Patchouli book, which you get when you earn a Forgero advancement.

**Reply by thecloudking** [reply]
patchouli isn't mentioned as being necessary on the modrinth description or version download pages

**Reply by SigmundGranaas** [reply]
It's supposed to be labelled as an optional dependency, but it seems like it is not present on Modrinth page. We should add it there as well like we have on CurseForge. Also, we should probably include our other compat mod in the optional section, like BetterCombat, LambDynLights, Mythic Metals and better nether.

**Comment by SigmundGranaas** [comment]
We should probably work on a dedicated online wiki or something. I don't know when we'll have time for it, but maybe someday someone will do it.

**Reply by Steveplays28** [reply]
Would be happy to set one up when I'm not modding the game

**Reply by CozyWithAsh** [reply]
I'd be more than happy to help! I've already made a small list on Google Sheets of some stuff. If that's something that's still of interest, lmk!
